lib/jekyll/configuration.rb

```diff
@@ -269,21 +269,18 @@ def add_default_excludes
 
     private
 
+    STYLE_TO_PERMALINK = {
+      :date     => "/:categories/:year/:month/:day/:title:output_ext",
+      :none     => "/:categories/:title:output_ext",
```

Comment:
You could move this up one line so that all `/:categories/:year/...` values are clubbed together.

Reply:
I did it this way because now the keys are in alphabetical order. It is easier to find out what the keys are.
Do you think that the alphabetical order of the values ​​make more sense?

Reply:
I would personally prefer a mix of both. `:none` as in denoting no date parameters followed by ones involving the date attribute in alphabetic order:
```ruby
STYLE_TO_PERMALINK = {
  :none     => "/:categories/:title:output_ext",
  :date     => "/:categories/:year/:month/:day/:title:output_ext",
  :ordinal  => "/:categories/:year/:y_day/:title:output_ext",
  :pretty   => "/:categories/:year/:month/:day/:title/",
  :weekdate => "/:categories/:year/W:week/:short_day/:title:output_ext",
}.freeze
```
That said, I don't have a strong preference on this either ways.